cmd/configmap-server/main.go

```diff
@@ -8,14 +8,14 @@ import (
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"google.golang.org/grpc"
+	health "google.golang.org/grpc/health/grpc_health_v1"
```

Comment:
We had a copy of this package in our repo, and some dependencies pulled in from this commit caused a [namespace conflict](https://protobuf.dev/reference/go/faq/#namespace-conflict) because two different packages defined the same GRPC protobuf names: our copy, and this official package.

To resolve, I deleted our package and switched everything to the official package.